doc/nrf/libraries/networking/nrf_cloud_log.rst

```diff
@@ -17,7 +17,7 @@ Overview
 ********
 
 This library provides an API for REST-, MQTT-, or CoAP-based applications to send logs to nRF Cloud.
-For MQTT- and CoAP-based applications, you can enable or disable logging as well as change the logging level remotely using the nRF Cloud portal or `nRF Cloud Patch Device State`_ REST API.
+For MQTT- and CoAP-based applications, you can enable or disable logging as well as change the logging level remotely using the nRF Cloud portal or `nRF Connect for Cloud Patch Device State`_ REST API.
```

Comment:
The original link is available in the links.txt file, better and works. We should actually get rid of any links with the old product name "nRF Connect for Cloud...".